pkg/driverbuilder/docker.go

```diff
@@ -284,39 +283,19 @@ func (bp *DockerBuildProcessor) Start(b *builder.Build) error {
 }
 
 func copyFromContainer(ctx context.Context, cli *client.Client, ID, from, to string) error {
-	rc, _, err := cli.CopyFromContainer(ctx, ID, from)
+	content, stat, err := cli.CopyFromContainer(ctx, ID, from)
 	if err != nil {
 		return err
 	}
-	defer rc.Close()
+	defer content.Close()
 
-	tr := tar.NewReader(rc)
```

Comment:
Thanks for this!

IIUC the change here is to replace usage of the `archive/tar` module with `github.com/docker/docker/pkg/archive` to handle `tar`-related operations.

Should we also do similarly in [`tarWriterFiles`](https://github.com/falcosecurity/driverkit/blob/aca2c0717b6280878ac81112a47e9b0aabefdf70/pkg/driverbuilder/docker.go#L317), then remove the import of the archive/tar module altogether?

Reply:
Uh nice, i will give it a look!

Reply:
I gave it a try; it seems not trivial to use same method for `tarWriterFiles` to get rid of it, because we copy multiple files from in-memory strings; if i use the "new" method, it will complain that the destination folder (that is: "/driverkit/") does not exist.